### ISSUE TITLE
Add list of relevant geo-gazetteer authority IDs

### ISSUE THREAD

**Issue by kintopp** [culturesofknowledge/emplaces#7]
Which are included in GeoNames? Are these represented correctly in the sample RDF? Do we want to add more (and if so, where will we find them)?

**Comment by kintopp** [comment]
Current mockup sees these appearing in the core data section, under 'See also:'

**Comment by gklyne** [comment]
The design currently calls for `em:source` in the data for non-GeoNames gazetteers.  `em:coreDataRef` is declared to be a subproperty of `em:source`, used for the GeoNames core.  

I need to check this is reflected in the data.  So far, I'm not aware of alternative gazetteer references available from the GeoNames data, but I could be missing something.  This could get fleshed out when start on wrangling the GeoNames data.

**Comment by gklyne** [comment]
This is all re-worked in the multi-source version of the data model.

Can we close this issue now?

**Comment by kintopp** [comment]
Agreed, the original discussion point at the top of the thread is resolved. I'll leave the issue open as a general placeholder for future authority related discussions.

**Comment by kintopp** [comment]
Just as a comment – it's a good thing that we now support multi-source reference gazetteers. I've realised that without this, people using EM Dates would be forced to first match their location references to either GeoNames or EM Places. That's a barrier for someone who, for example, wants to convert date metadata with accompanying locations previously linked against e.g. German GND or French BNF authorities. We'll need to find the matching authority IDs for our GeoName and then our own references. Possibly via Getty TGN, else possibly Who's On First https://www.whosonfirst.org Later, also to http://whgazetteer.org.

**Comment by kintopp** [comment]
The idea is that if someone provides an e.g. GND or BNF or Getty TGN place reference to EM Dates, a query to EM Places will show if we know the ID, and so be able to look for the appropriate historical entity and related calendrical data. In the case of an EM Places ID (or, a WHG ID) we'd be able to look-up the entity directly.
